sopel/modules/xkcd.py

```diff
@@ -121,20 +123,26 @@ def numbered_result(bot, query, latest, commanded=True):
 
 
 def say_result(bot, result, commanded=True):
-    message = '{}{} | Alt-text: {}'.format(
-        result['url'] + ' | ' if commanded else '',
-        result['title'], result['alt']
-    )
-    bot.say(message)
+    parts = []
 
+    if commanded:
+        parts.append(result['url'])
```

Comment:
It would be even more consistent if the `url` part was appended last.

GitHub has decreed that I may not make a single suggestion that would do it for you, though, since it would cross over a dElETeD LiNe and that's forbidden. So I'm afraid you'll have to just edit the file yourself if you like this idea. Sorry. 😸